### ISSUE TITLE
Daz to Blender Add On not in Blender

### ISSUE THREAD

**Issue by MSG0120** [daz3d/DazToBlender#110]
I have installed Daz to Blender into Daz 3D.  It shows up in the Scripts menu. I ran the Script on a Daz model and it appeared to be created with the script.  When I go into Blender (3.0.0 on a PC), and search for Daz under Add Ons on the Preferences panel, nothing shows up.  What else do I need to do to get the Daz to Blender Add On added to Blender?

**Comment by rafed1973** [comment]
Certainly, there are compatibility issues with Blender 3.0 and beyond.
There is a way to get the DAZ to Blender add-on to show in bender 3.0 but even if you do it will not import the DAZ models correctly and it will give you some errors as well.
We are waiting for an update from the creator.

**Comment by MSG0120** [comment]
Thank you so much.
________________________________

**Comment by rafed1973** [comment]
You are most welcome for the time being I kept my 2.93 version of Blender just in case I wanted to do any DAZ work on it.
Cheers!
